code/modules/mob/living/carbon/xenomorph/castes/queen/queen.dm

```diff
@@ -227,3 +229,13 @@
 		set_queen_overwatch(observed_xeno, TRUE)
 	if(ovipositor)
 		dismount_ovipositor(TRUE)
+
+
+// ***************************************
+// *********** Larva Mother
+// ***************************************
+
+/mob/living/carbon/xenomorph/queen/proc/is_burrowed_larva_host()
+	if(!ovipositor || incapacitated(TRUE))
+		return COMSIG_HIVE_XENO_MOTHER_FALSE
```

Comment:
flip this to return the truthful signal if it is a host and dont return anything otherwise then just 
send_signal() & truthful